py3status/modules/xrandr_rotate.py

```diff
@@ -105,6 +105,7 @@ def _switch_selection(self):
 
     def xrandr_rotate(self):
         all_outputs = self._get_active_outputs()
+        screen = self.screen if self.screen else None
```

Comment:
This looks wrong to me. I'm on my phone so can't do a proper review.

```
screen = self.screen or None
```

Would be cleaner but I still am not happy. If I see more of the code it might make sense.

Reply:
`psc` check is old news due to possible laptop docking and/or monitors plugging in/out... and having `psc` check still there would always prevent `COLOR_DEGRADED` from happening.

I use `screen = self.screen if self.screen else None` instead.
I will use your cleaner suggestion above.